### PR TITLE
Implemented API versioning

### PR DESCRIPTION
**Pull request by Aryan-sharma11**
Hey,  this pull request is regarding https://github.com/biosharp-dotnet/sucuri-api/issues/19 

- Right now I have restructured the Private folder to store scripts for the different versions of API 

- Now we can use version 1 using  `..\v1\search?pkg_name=<package_name>`

Please, suggest any changes  and improvements that are needed

Kindly, Aryan

### CODE REVIEW

**Review by abhi18av — CHANGES_REQUESTED** [review]
Thanks @Aryan-sharma11 , this is looking good.


The OpenAPI spec is reflecting the changes ✅ 

image


Just one minor change regarding the doc message which needs to be improved - it should include the link for OpenAPI as well

image

**Comment by Aryan-sharma11** [comment]
> Thanks @Aryan-sharma11 , this is looking good.
> 
> The OpenAPI spec is reflecting the changes ✅
> 
> image
> 
> Just one minor change regarding the doc message which needs to be improved - it should include the link for OpenAPI as well
> 
> image

@abhi18av I have addressed this issue in https://github.com/biosharp-dotnet/sucuri-api/pull/25

File: Sucuri-API/Public/Start-SucuriServer.ps1 [hunk 1]
```diff
@@ -51,11 +51,13 @@ function Start-SucuriServer {
 
       ##########  Version 1 ##########
 
+      $serverName = "$env:COMPUTERNAME:8085/docs/v1"
```
Comment (abhi18av) [inline Sucuri-API/Public/Start-SucuriServer.ps1:54]:
```suggestion
      $serverName = "$env:COMPUTERNAME:8085/docs/v1"

```
This does not exist on non-windows machine - let's not change this.

File: Sucuri-API/Public/Start-SucuriServer.ps1 [hunk 2]
```diff
@@ -51,11 +51,13 @@ function Start-SucuriServer {
...
 
+      $serverName = "$env:COMPUTERNAME:8085/docs/v1"
+
       #Enabling OpenApi specifications
-      Enable-PodeOpenApi -Path '/docs' -Title 'Sucuri-API' -Version 0.9 -Description 'Provides latest version number of any given conda package'`
+      Enable-PodeOpenApi -Path '/docs/v1' -Title 'Sucuri-API' -Version 0.9 -Description 'Provides latest version number of any given conda package'`
       -RestrictRoutes '/v1/search'
 
-      Enable-PodeOpenApiViewer -Type Swagger -Path '/docs/view' -DarkMode
+      Enable-PodeOpenApiViewer -Type Swagger -Path '/docs/v1/view' -OpenApiUrl $serverName -DarkMode
```
Comment (abhi18av) [inline Sucuri-API/Public/Start-SucuriServer.ps1:61]: @Aryan-sharma11 , let's keep the `docs` endpoint away from `v1/v2` etc.

**Review by abhi18av — CHANGES_REQUESTED** [review]
Requested some changes - but nothing breaking.

**Pull request merged**